.github/workflows/node.js.yml

```diff
@@ -124,7 +124,7 @@ jobs:
       - uses: garygrossgarten/github-action-ssh@release
         name: Run the Update MicroFronEnd Script
         with:
-            command: cd /usr/share/tomcat/microfrontends/working && sh update_microfrontends.sh
+            command: cd /usr/share/tomcat/microfrontends/working && node microfrontends.release.js --src esm_packages.json  --type demo --dest microfrontends.json && /bin/bash update_microfrontends.sh
```

Comment:
@larslemos isn't type supposed to be 'working' since this is going to working server?

Reply:
Yes, fixed